enclave_manager/avalon_enclave_manager/kme/kme_enclave_info.py

```diff
@@ -115,10 +117,16 @@ def _create_signup_info(self, ias_nonce, config):
 
         signup_cpp_obj = enclave.SignupInfoKME()
 
+        if "wpe_mrenclave" in config:
+            self._wpe_mrenclave = config["wpe_mrenclave"]
+        else:
+            self._wpe_mrenclave = hex_utils.mrenclave_hex_string(
+                enclave_info.TCF_HOME + "/"
+                + config["wpe_mrenclave_read_from_file"])
         # @TODO : Passing in_ext_data_signature as empty string "" as of now
```

Comment:
Supposed to be for future use.

Reply:
@manju956 can you conform if we need "in_ext_data_signature " for KME. What's pending to be implemented in this release? If nothing is remaining we can remove above TODO comment

Reply:
@manojgop  we haven't considered in_ext_data_signature for KME signup flow for this release.
Because we are not aware of how ext_data is signed, what key is used and how to share the key during signup flow. It could be taken during 0.7 release.

Reply:
@manju956 Which key are you referring to here ? Why do we need `in_ext_data_signature` for KME signup ? Please elaborate ?

Reply:
`in_ext_data_signature` is added security to check integrity of in_ext_data (WPE's mrenclave value).
We don't have mechanism to share public part of signing key used to sign ext_data during KME boot flow.